### PR TITLE
Issue 2 view entries

### PR DESCRIPTION
**Pull request by Jktanisaka**
issue2donemaybe
issue2doneasdf

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,12 +50,26 @@ <h1>New Entry</h1>
...
+      <div class="row">
+        <div class="column-full flex align-center justify-between">
+          <h1>Entries</h1>
+          <a href="#" class="new-button text-center flex justify-center align-center" id="newButton">NEW</a>
+        </div>
+      </div>
+      <ul id="entriesList">
+        <li class="row no-entries text-center" id="noClass">
+          <p class="column-full p-no-entry">No entries have been recorded.</p>
+        </li>
```
Comment (codingcodymiller) [inline index.html]: This shouldn't be an `li`, it's not an entry in this list. I would just move this `p` element to be a sibling of the `ul`

File: js/main.js [hunk 2]
```diff
@@ -24,4 +24,70 @@ function submitForm(event) {
...
+}
+var noClass = document.querySelector('#noClass');
+var ul = document.querySelector('#entriesList');
+
+var savedArray = [];
+var savedEntries = localStorage.getItem('javascript-local-storage');
+
+if (savedEntries !== null) {
+  savedArray = JSON.parse(savedEntries);
+}
```
Comment (codingcodymiller) [inline js/main.js]: What is the purpose of this? All localStorage-related code should be in the `data.js` file, where the `data` variable is located. And you shouldn't be making your own variables to keep track of the entries, you should make use of the `data.entries` array

File: js/main.js [hunk 3]
```diff
@@ -24,4 +24,70 @@ function submitForm(event) {
...
+
+newButton.addEventListener('click', viewSwapMain);
+function viewSwapMain(event) {
+  mainView.setAttribute('class', 'container gray');
+  entriesView.setAttribute('class', 'container gray hidden');
+}
+
+function createEntries(event) {
+  for (var i = 0; i < savedArray.length; i++) {
+    JSON.parse(savedEntries);
```
Comment (codingcodymiller) [inline js/main.js]: This line isn't doing anything, it can be deleted.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Jktanisaka** [comment]
code journal issue 2
Updated localstorage and view switch function

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**